### PR TITLE
36 fix the rest of vulnerabilities

### PR DESCRIPTION
**Pull request by juanjoseortega-aircury**
To check "Code Scanning" fixes, you need to go into "Checks" on this PR, into "Code Scanning Results", click on "CodeQL" and now on "View all branch alerts.
Also can be checked going into "Security", "Code Scanning" and filtering by this pr. "pr:81"
Dependabot vulnerabilities can't be checked here.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,8 +2,9 @@ version: '2'
 
 services:
   jekyll:
-    image: jekyll/jekyll:4.2.0
-    command: jekyll serve --watch --force_polling --verbose --livereload
+    image: jekyll/jekyll:4.2.2
```
Comment (cristobaldiaz-aircury) [inline docker-compose.yml:5]: Do we want to set the same version that we have in the Gemfile?

File: Gemfile [hunk 2]
```diff
@@ -12,12 +12,12 @@ source "https://rubygems.org"
...
 # uncomment the line below. To upgrade, run `bundle update github-pages`.
 # all supported plugins by GitHub Pages are included in "github-pages". (GitHub Pages doesn't support 3rd party plugins)
-gem "github-pages", "~> 228"
+gem "github-pages", "~> 229"
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 # and associated library.
 install_if -> { RUBY_PLATFORM =~ %r!mingw|mswin|java! } do
-  gem "tzinfo", "~> 1.2"
+  gem 'tzinfo', '~> 2.0', '>= 2.0.6'
```
Comment (cristobaldiaz-aircury) [inline Gemfile:20]: Why this switch from double to single quotation mark?

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (juanjoseortega-aircury) [reply Gemfile:20]: This is recommended at rubygems page to install tzinfo gem.

File: docker-compose.yml (hunk 1 above)
Comment (juanjoseortega-aircury) [reply docker-compose.yml:5]: This version is from the docker repository of a contributor, this isn't the version of jekyll, but this docker uses the version that is set on the gemfile.

**Review by cristobaldiaz-aircury — APPROVED** [review]

**Pull request merged**